data/brands/tourism/hotel.json

```
@@ -18,6 +18,16 @@
     }
   },
   "items": [
+     {
+      "displayName": "ayenda",
+      "locationSet": {"include": ["co","pe","mx"]},
+      "tags": {
+        "brand": "ayenda",
+        "brand:wikidata": "Q122723982",
+        "name": "ayenda",
```

```suggestion
      "displayName": "Ayenda",
      "locationSet": {"include": ["co","pe","mx"]},
      "tags": {
        "brand": "Ayenda",
        "brand:wikidata": "Q122723982",
        "name": "Ayenda",
```
Title case is [generally preferred](https://wiki.openstreetmap.org/wiki/Names#Uppercase_characters) for names.